web_responsive/views/web.xml

```diff
@@ -0,0 +1,242 @@
+                                <i class="fa fa-th fa-lg app-drawer-icon-open" />
+                            </a>
+
+                            <button type="button"
+                                    class="app-drawer-toggle drawer-toggle pull-left navbar-toggle collapsed"
+                                    >
+                                <span class="sr-only">Toggle App Drawer</span>
+                                <div class="fa fa-th fa-lg app-drawer-icon-open" />
+                            </button>
+
+                            <button type="button"
+                                    id="odooMenuBarToggle"
+                                    class="navbar-toggle collapsed pull-right"
+                                    data-toggle="collapse"
+                                    data-target="#odooMenuBarNav"
+                                    >
+                                <span class="sr-only">Toggle Navigation</span>
+                                <i class="fa fa-bars fa-lg" />
+                            </button>
+
+                        </div>
+
+                        <div class="collapse navbar-collapse text-center"
+                             id="odooMenuBarNav"
+                             data-parent="#odooMenuBarToggle"
+                             aria-expanded="false"
+                             >
+                            <div class="oe_leftbar"
+                                 groups="base.group_user,base.group_portal"
+                                 >
+                                <t t-call="web.menu_secondary" />
+                            </div>
+                            <div class="nav navbar-nav navbar-right">
+                                <ul class="nav navbar-nav navbar-right oe_user_menu_placeholder"
+                                    style="display: none;"
+                                    />
+                                <ul class="nav navbar-nav navbar-right oe_systray"
+                                    style="display: none;"
+                                    />
+                            </div>
+                        </div>
+                    </div>
+                </nav>
+
+            </header>
+
+        </xpath>
+
+    </template>
+
+    <template id="menu_secondary"
+              inherit_id="web.menu_secondary"
+              name="App Drawer - Secondary Menu"
+              >
+
+        <xpath expr="//a[@class='oe_logo']" position="replace" />
+
+        <xpath expr="//div[@class='oe_footer']" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+
+        <xpath expr="//div[@class='oe_secondary_menus_container']/t" position="replace">
+
+            <t t-foreach="menu_data['children']" t-as="menu">
+                <ul style="display: none" class="oe_secondary_menu nav navbar-nav" t-att-data-menu-parent="menu['id']">
```

Comment:
This is copied from Odoo core, and is required for their Javascript unfortunately